src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -740,6 +742,7 @@ public static IEnumerable<InstructionSetInfo> ArchitectureToValidInstructionSets
                     yield return new InstructionSetInfo("Vector64", "", InstructionSet.ARM64_Vector64, false);
                     yield return new InstructionSetInfo("Vector128", "", InstructionSet.ARM64_Vector128, false);
                     yield return new InstructionSetInfo("Dczva", "", InstructionSet.ARM64_Dczva, false);
+                    yield return new InstructionSetInfo("rcpc", "Rcpc", InstructionSet.ARM64_Rcpc, true);
```

Comment:
The ManagedName is the name of C# class that hold the intrinsics and has the `IsSupported` method. The code in HardwareIntrinsicHelpers.Aot.cs then makes sure calls to `IsSupported` read the value computed in PalRedhawkUnix.cpp at startup.

Since there's no managed class with this name, none of this code will work. Should the last parameter here be `false` instead and ManagedName empty, same as Dczva above?

Reply:
Thanks, fixed!